surah/src/commonMain/kotlin/com/islamversity/surah/SurahProcessor.kt

```diff
@@ -39,21 +40,46 @@ class SurahProcessor(
                         add(0, surah)
                     }
                 }
+                    .map {
+                        SurahResult.Items(it) to initial
+                    }
             }
-            .map {
-                SurahResult.Items(it)
-            }
+            .publish(
+                {
```

Comment:
Is this the responsibility of the processor or usecase?

Reply:
I'm trying to send an event to move the list to the required item.
This is entirely related to UI and no business logic is involved
What that said, I can say this is a processor job to create the view state